cli/create.go

```diff
@@ -259,6 +259,16 @@ func createContainer(ociSpec oci.CompatOCISpec, containerID, bundlePath,
 		return vc.Process{}, err
 	}
 
+	// Add the ephemeral device if ephemeral volume
+	// has to be attached to the container
+	for _, mnt := range contConfig.Mounts {
+		if IsEphemeralStroage(mnt.Source) {
+			deviceInfo := config.DeviceInfo{}
+			deviceInfo.DevType = "e"
```

Comment:
It would be more explicit to have something like `ephemeral` instead of `e`, it would read better.
Also, whatever value we choose, we need to reference this from a constant or a global variable to be reused across all virtcontainers.

Reply:
@sboeuf I agree that it should use a full word instead of just a letter. However, all other devices use deviceInfo.DevType which is declared [here](https://github.com/kata-containers/runtime/blob/master/virtcontainers/device/config/config.go#L63) single letter. So just for ephemeral if I use a word that would deviate from the standard other device types are following. 

I spoke about this with @bergwolf . We agreed that in future we might have to change that for all devices, but that would be out of scope for this PR.

Reply:
Yes, fair enough !